src/styles/main.scss

```diff
@@ -0,0 +1,43 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
+  padding-inline: $cellPading;
```

Comment:
There is a typo in the variable `$cellPading`; it should be `$cellPadding`. Correct spelling is crucial for code readability and to avoid any confusion.